tron/core/actionrun.py

```diff
@@ -429,6 +429,7 @@ def retry(self):
         if self.in_delay is not None:
             self.in_delay.cancel()
             self.in_delay = None
+        self.retries_delay = None
```

Comment:
i think this is going to affect subsequent automatic retries. do we care? if yes - we should instead have something like `self.skip_next_retries_delay = True` and relevant logic.

Reply:
Hmmm I think this solution is good enough (even though it did irritate my idealism a little bit at first :) )

If users are manually retrying an action, then there probably aren't any automatic retries remaining, so the retry delay doesn't really matter anymore.

Reply:
Yea I didn't think we cared at this point if a human is running the `retry` command.
y/fixit